cli/status.go

```diff
@@ -55,8 +55,8 @@ func statusCommand(t *core.Timetrace) *cobra.Command {
 			if format != "" {
 				format = strings.ReplaceAll(format, "{project}", project)
 				format = strings.ReplaceAll(format, "{trackedTimeCurrent}", trackedTimeCurrent)
-				format = strings.ReplaceAll(format, "{todayTime}", t.Formatter().FormatTodayTime(report))
-				format = strings.ReplaceAll(format, "{breakTime}", t.Formatter().FormatBreakTime(report))
+				format = strings.ReplaceAll(format, "{trackedTimeToday}", t.Formatter().FormatTodayTime(report))
+				format = strings.ReplaceAll(format, "{breakTimeToday}", t.Formatter().FormatBreakTime(report))
```

Comment:
You also need to change the Flags available fields.

Reply:
Thanks for pointing this out!